lib/services/swagger-api-service.js

```diff
@@ -105,10 +109,23 @@ function swaggerFactory(
         };
     }
 
+    function swaggerRenderer(templateName, data) {
+        return templates.get(templateName, ['global'])
+            .then(function(template) {
+                return template.contents;
+            })
+            .then(function(contents) {
+                var output = Promise.map(data, function(item) {
+                    return JSON.parse(ejs.render(contents, item));
+                });
+                return output;
+            });
```

Comment:
@dalebremner Should be a `.catch(error)` here that calls `next(_processError(err)`